internal/api/v1/organizations.go

```diff
@@ -170,8 +160,10 @@ func (oc NamespacesController) Create(w http.ResponseWriter, r *http.Request) AP
 		return InternalError(err)
 	}
 
+	w.Header().Set("Content-Type", "application/json")
```

Comment:
isn't `jsonResponse` setting this, 2 lines below?

Reply:
Haha, great that you found that.
Yes, but this method is one of the few exceptations, that want to return their own header.  In the next line:

>  `w.WriteHeader(http.StatusCreated)`

So, I read the API docs for the http funcs and I think this way we return both, the content type and the 201 status code. The `SetHeader` in `jsonResponse` will be ineffective and the call to `r.Write` will not send another header.

I would suggest to add the status code to `jsonResponse` (or it's replacement) in the future.

Reply:
This is hard to follow. So, any `w.Header().Set()` that comes after an explicit `w.WriteHeader` will also have no effect? Then I wonder why we can still write the body (inside the `jsonResponse` method).

In any case, I think you are right, we should pass the status code to `jsonResponse`. I think the response object should have one more field for that.